applications/nrf_desktop/Kconfig.debug

```diff
@@ -0,0 +1,174 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config DESKTOP_LOG
+	bool "nRF Desktop logging"
+	help
+	  The option enables support for the nRF Desktop Logging.
+	  By default, the nRF Desktop Logging uses the RTT backend and
+	  slightly modifies the default configuration of the Logging
+	  subsystem to tailor it to the application needs.
+
+if DESKTOP_LOG
+
+config LOG
+	default y
+	help
+	  nRF Desktop Logging configuration enables the Logging subsystem that
+	  is required for this functionality.
+
+choice LOG_MODE
+	default LOG_MODE_DEFERRED
+	help
+	  nRF Desktop Logging uses the deferred logging mode to minimize the
+	  impact of the debug tools on the application.
+
+endchoice
+
+config LOG_DEFAULT_LEVEL
+	default 2
+	help
+	  nRF Desktop Logging configuration sets the Warning level as a default
+	  logging level for modules that do not specify their own logging level.
+
+config LOG_MODE_OVERFLOW
+	default y
+	help
+	  nRF Desktop Logging configuration drops the oldest logs if there is
+	  no more space left in the buffers.
+
+config LOG_PRINTK
+	default y
+	help
+	  nRF Desktop Logging configuration enables the processing of printk
+	  messages by the Logging subsystem.
+
+config LOG_BUFFER_SIZE
+	default 4096
+	help
+	  nRF Desktop Logging configuration increases the default buffer
+	  size in the Logging subsystem to prevent logs from dropping.
+
+config LOG_PROCESS_THREAD_STACK_SIZE
+	default 1024
+	help
+	  nRF Desktop Logging configuration increases the default stack size
+	  for the log processing thread to prevent stack overflows.
+
+config LOG_BACKEND_RTT
+	default y if !SHELL_BACKEND_RTT
```

Comment:
if !LOG_BACKEND_SHELL?

Reply:
That would also be more consistent with help and doc

Reply:
do you mean CONFIG_SHELL_LOG_BACKEND?

Reply:
fixed